lnrpc/rpc.proto

```diff
@@ -1381,6 +1389,19 @@ message GetInfoResponse {
     map<uint32, Feature> features = 19;
 }
 
+message GetRecoveryInfoRequest {
+}
+message GetRecoveryInfoResponse {
+    // Whether the wallet is in recovery mode
+    bool recovery_mode = 1;
+
+    // Whether the wallet recovery progress is finished
+    bool recovery_finished = 2;
+
+    // The recovery progress
+    double progress = 3;
```

Comment:
We should note that this is between 0 and 1.

Reply:
yep, fixed.